src/components/Konnector.jsx

```diff
@@ -13,10 +13,11 @@ import { getTriggersByKonnector } from 'reducers'
 export const Konnector = ({ konnector, history, triggers }) => {
   const konnectorWithTriggers = { ...konnector, triggers: { data: triggers } }
   const onDismiss = useCallback(() => history.push('/connected'), [history])
+  const slug = konnector?.slug || location.hash.split('/')[2]
```

Comment:
This would return undefined if the url path does not contain enough `/`. Then the konnectorRoot would equal `/connected/undefined`. Maybe we should handle that scenario?

Reply:
https://github.com/cozy/cozy-home/pull/1753/files#diff-5effd6f1a25891c92d544d8648f957ebf6bd2f28b259be0bec3164bcfaecbfd0R19